src/recon_buildblock/BinNormalisationFromECAT8.cxx

```diff
@@ -282,10 +282,13 @@ MatrixFile* mptr = matrix_open(filename.c_str(),  MAT_READ_ONLY, Norm3d);
   s.erase( std::remove_if( s.begin(), s.end(), isspace ), s.end() );
   /*interfile_parser.*/data_file_name=s;
   
-  if (/*interfile_parser.*/originating_system == "2008")
+  if (/*interfile_parser.*/originating_system == "2008"
+      or originating_system == "SiemensmMR" // we stripped the space
```

Comment:
```suggestion
      || (originating_system == "SiemensmMR") // we stripped the space
```
C++11? fails on [Appveyor](https://ci.appveyor.com/project/KrisThielemans/stir/builds/20076321/job/kw1tosgov218mcbn)
I feel safer with `()`
similar elsewhere.

Reply:
by the way, why stripped the space?

Reply:
Haha, ah can you tell what language I've been using all day?

The stripped space was done a few lines prior, apparently to remove trailing newline (specifically \r).